### DISCUSSION TITLE
Is this image available on ECR?

### DISCUSSION THREAD

**Discussion by HappyNinja2** [tiangolo/uwsgi-nginx-docker#224] [Questions] [answered]
Using AWS code pipeline, it's not possible to download the docker image all the time, because Docker site blocks AWS IPs and rate limits them. Is this image available on AWS ECR?

**Comment by tiangolo** [comment, marked as answer]
No, this image is currently available in Docker Hub. Although you could pull and push it to your ECR instance.
